### ISSUE TITLE
Transition animation not working on android and WP

### ISSUE THREAD

**Issue by xap5xap** [driftyco/ionic#7171]
The animation "ios-transition" causes the returning page to occupy half of the screen when the back button is pressed. This only happens on Android and WP.

**Steps to reproduce:**
When I open another page with the code below, the animations works as expected from right to left on IOS, Android and WP, but when the back button is pressed on the opened page, then the parent’ content is rendered only the half.

```
        this.navController.push(SearchPage, null, { animate: true, animation: "ios-transition" });      
```
 
![image](https://cloud.githubusercontent.com/assets/1402054/16541278/4d88a248-4044-11e6-8c91-efa4827ed07f.png)

I am using Ionic 2.0.0-beta.10

Cordova CLI: 6.1.1
Gulp version:  CLI version 3.9.1
Gulp local:
Ionic CLI Version: 2.0.0-beta.32
Ionic App Lib Version: 2.0.0-beta.18
OS:
Node Version: v4.4.6

**Comment by manucorporat** [comment]
Confirmed this is a issue in our e2e tests:
<img width="605" alt="screen shot 2016-07-03 at 09 38 46" src="https://cloud.githubusercontent.com/assets/127379/16544326/1e683cbc-4102-11e6-8b35-59d26ef83dde.png">

**Comment by gadld** [comment]
Hi, was this fixed? because I'm running beta.11 on ios 9.3.2 and still getting this issue

**Comment by schankam** [comment]
Still getting the issue as well. Didn't see it happen on Android though, but on iOS.

**Comment by manucorporat** [comment]
@schankam @gadld should be fixed thanks to: #8272
